@here/harp-mapview/lib/MapView.ts

```diff
@@ -1261,6 +1261,13 @@ export class MapView extends EventDispatcher {
         return this.m_tileWrappingEnabled;
     }
 
+    set tileWrappingEnabled(enabled: boolean) {
+        if (enabled !== this.m_tileWrappingEnabled) {
+            this.m_tileWrappingEnabled = enabled;
+            this.m_visibleTiles = this.createVisibleTileSet();
```

Comment:
You should check if the Projection is Planar before creating an new tile set, for the spherical projection the tileWrapping property should not change anything

Reply:
Opps, missed your comment, thanks, I will fix!